src/Mod/Image/ImageTools/_CommandImageScaling.py

```diff
@@ -37,31 +37,14 @@
 
     import FreeCADGui
 
-# Translation-related code
-# See forum thread "A new Part tool is being born... JoinFeatures!"
-# http://forum.freecadweb.org/viewtopic.php?f=22&t=11112&start=30#p90239
-    try:
-        _fromUtf8 = QtCore.QString.fromUtf8
-    except (Exception):
-        def _fromUtf8(s):
-            return s
-    try:
-        _encoding = QtGui.QApplication.UnicodeUTF8
-        def _translate(context, text, disambig):
-            return QtGui.QApplication.translate(context, text, disambig, _encoding)
-    except (AttributeError):
-        def _translate(context, text, disambig):
-            return QtGui.QApplication.translate(context, text, disambig)
-
-
 # command class
 class _CommandImageScaling:
     "Command to Scale an Image to an Image Plane"
     def GetResources(self):
         return {'Pixmap': "Image_Scaling",
-                'MenuText': QtCore.QT_TRANSLATE_NOOP("Image_Scaling", "Scale image plane"),
+                'MenuText': QtGui.QApplication.translate("Image_Scaling", "Scale image plane", None),
                 'Accel': "",
-                'ToolTip': QtCore.QT_TRANSLATE_NOOP("Image_Scaling", "Scales an image plane by defining a distance between two points")}
+                'ToolTip': QtGui.QApplication.translate("Image_Scaling", "Scales an image plane by defining a distance between two points", None)}
```

Comment:
This is incorrect: menu text and tooltips should always use the NOOP, the actual translation lookup occurs elsewhere.